src/js/pages/services/ServicesTab.js

```diff
@@ -103,6 +102,20 @@ var ServicesTab = React.createClass({
     this.setState(state, this.resetFilterQueryParams);
   },
 
+  getAlertPanelFooter: function () {
+    return (
+      <div className="button-collection flush-bottom">
+        <button className="button button-stroke button-inverse"
+          onClick={this.handleOpenGroupFormModal}>
```

Comment:
unfortunately, i believe the modal will never ever show, even if this is clicked. this is because we render the modal in `getServiceTreeView` (which we don't call if there are empty services).

this fix to this to render the modal always in `#render` right next to `#getContents`